performance/pom.xml

```diff
@@ -245,7 +245,7 @@
             <properties>
                 <validation-api.version>1.1.0.Final</validation-api.version>
                 <beanvalidation-impl.name>Hibernate Validator</beanvalidation-impl.name>
-                <beanvalidation-impl.version>5.4.2.Final</beanvalidation-impl.version>
+                <beanvalidation-impl.version>5.4.3.Final</beanvalidation-impl.version>
             </properties>
             <dependencies>
                 <dependency>
```

Comment:
Because it's using BV 2 constraints :).

I had an issue when trying to build with BV 1.1 and couldn't find nice BV 1.1 constraints so I just decided to move it as it's more a test to validate recent performance improvements.